### ISSUE TITLE
There are no instructions on Installing WAC on WS

### ISSUE THREAD

**Issue by walterov** [MicrosoftDocs/windowsserverdocs#6487]
Is this page meant to show how to install WAC on WS? The title seems to point to it, yet there are no instructions. What's the purpose of this page?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 49b4fd75-78b8-2a23-f908-9b048f04d831
* Version Independent ID: ff42b689-4845-e9f0-d124-292b614fd016
* Content: [Install Windows Admin Center](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/deploy/install)
* Content Source: [WindowsServerDocs/manage/windows-admin-center/deploy/install.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/main/WindowsServerDocs/manage/windows-admin-center/deploy/install.md)
* Product: **windows-server**
* Technology: **windows-admin-center**
* GitHub Login: @jwwool
* Microsoft Alias: **jeffrew**

**Comment by milldogtjm** [comment]
@walterov You are right that this page does not and that is cause this is just a general overview page on how to install the Windows Admin Center to an environment. So there for the the answer is that the install instructions and advanced options are on https://learn.microsoft.com/en-us/windows-server/manage/windows-admin-center/plan/installation-options to give a more Indepth instruction to install this to either a client or to a server.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
